tsdb/chunks/chunks.go

```diff
@@ -190,40 +190,50 @@ func (w *Writer) cut() error {
 		return err
 	}
 
-	p, _, err := nextSequenceFile(w.dirFile.Name())
+	n, f, _, err := cutSegmentFile(w.dirFile, chunksFormatV1, true, w.segmentSize)
 	if err != nil {
 		return err
 	}
+	w.n = int64(n)
+
+	w.files = append(w.files, f)
+	if w.wbuf != nil {
+		w.wbuf.Reset(f)
```

Comment:
No need for `else` here - just return is enough to simplify.

Reply:
It is required :) only one of them should be run